build/root/WORKSPACE

```diff
@@ -58,6 +58,14 @@ http_file(
     urls = mirror("https://storage.googleapis.com/kubernetes-release/network-plugins/cni-plugins-amd64-v0.6.0.tgz"),
 )
 
+CRI_TOOLS_VERSION = "1.0.0-beta.1"
+
+http_file(
+    name = "cri_tools",
+    sha256 = "bdc838174778223a1af4bdeaaed4bd266120c0e152588f78750fb86221677fb4",
+    urls = mirror("https://github.com/kubernetes-incubator/cri-tools/releases/download/v%s/crictl-v%s-linux-amd64.tar.gz" % (CRI_TOOLS_VERSION, CRI_TOOLS_VERSION)),
```

Comment:
great parameterization here, we actually get this wrong for CNI I see... :thinking:

Reply:
nah, i saw that too and checked it and it's actually right.

Reply:
unless you mean literally that we don't parameterize the cni -- the url is correct though